app/routes.php

```diff
@@ -228,6 +228,7 @@ public function register(string $idName = 'id')
         ->setAuthMask(true)
 		->register();
 	#FIXME ------ WTH is this endpoint? Makes no sense
+	// Note: This is an endpoint for parameters bound to a specific reaction, as opposed to global parameters - Havlík
```

Comment:
still dont get it. Ctl\ReactionItemParentedParameterController::class does not use this argument: reactionItem-id to get the reactionItem, but the reaction. So I guess this is just a typo.